samples/subsys/llext/modules/prj.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_LOG=y
+CONFIG_LOG_MODE_IMMEDIATE=y
+
+CONFIG_MODULES=y
```

Comment:
Nit, and definitely out of this PR's scope: I find this Kconfig option's name quite confusing at first glance.

Reply:
Indeed it is. We can't change it unless we modify `kconfiglib` internals, though, since that is the name that triggers the whole "tristate" stuff in Kconfig. :slightly_frowning_face: